src/hotspot/share/gc/z/zRelocate.cpp

```diff
@@ -382,7 +353,7 @@ zaddress ZRelocate::relocate_object(ZForwarding* forwarding, zaddress_unsafe fro
   ZForwardingCursor cursor;
 
   // Lookup forwarding
-  zaddress to_addr = forwarding_find(forwarding, from_addr, &cursor);
+  zaddress to_addr = forwarding->find(from_addr, &cursor);
```

Comment:
While looking at this, I now see that this could use the find overload that doesn't take a ZForwardingCursor. Could you update the code to do so?

Reply:
I agree. I missed that insert call in the diff.